_redirects

```diff
@@ -0,0 +1,40 @@
+/api/cucim /api/cucim/stable/
+/api/cucim/ /api/cucim/stable/
+/api/cudf /api/cudf/stable//
```

Comment:
@ajschmidt8 Was the issue you saw in this comment https://github.com/rapidsai/docs/pull/401#issuecomment-1589550633 just a typo?
```suggestion
/api/cudf /api/cudf/stable/
```

Reply:
ohhhhh shoot. yes. most likely.

i was doing bulk updates via some fancy VS Code keyboard shortcuts and they must've gone awry.

let me update this and try again.

good catch. this is why we do code reviews :slightly_smiling_face: